Gems/Atom/Feature/Common/Assets/Shaders/Materials/BasePBR/BasePBR_PixelGeometryData.azsli

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+
+#define PIXEL_GEOMETRY_DATA_IS_BASE_PBR     1
+
+struct PixelGeometryData
```

Comment:
Suggest `SurfelGeometry` perhaps since we can potentially imagine this data used for object-space shading in addition to the pixel shader use case used here.

For a future refactor, it may be nice to make this data structure tighter. I imagine, for example, that `isFrontFace` and the tangent frame is used mainly to compute a world-space normal, which would replace all the data here except for the uvs and world-space position (the `isFrontFace` check I'm assuming is used to flip the normal for back facing triangles that aren't culled). In this case, the code responsible for filling out this struct would simply compute the world-space normal and uvs needed, accounting for non-uniform TBN frames and triangle orientation.